### PR TITLE
ODBPrimitiveEncodingStream-define-subclassResponsibility

### PR DESCRIPTION
**Pull request by MarcusDenker**
- add #subclassResponsibility methods or push up methods from ODBMemoryStreamWrapper
- tag ODBPrimitiveEncodingStream as Abstract

### CODE REVIEW

File: src/OmniBase/ODBPrimitiveEncodingStream.class.st [hunk 1]
```diff
@@ -7,11 +7,28 @@ Class {
...
+]
+
 { #category : #accessing }
 ODBPrimitiveEncodingStream >> atEnd [ 
 	^ stream atEnd
 ]
 
+{ #category : #accessing }
+ODBPrimitiveEncodingStream >> flush [ 
+
```
Comment (noha) [inline src/OmniBase/ODBPrimitiveEncodingStream.class.st:29]: Ignoring a flush that is called is not a good idea. If someone needs to flush we need to delegate that to the underlying stream

Comment (MarcusDenker) [reply src/OmniBase/ODBPrimitiveEncodingStream.class.st]: fixed!

File: src/OmniBase/ODBFileStreamWrapper.class.st [hunk 2]
```diff
@@ -212,11 +198,7 @@ ODBFileStreamWrapper >> name [
 
 { #category : #accessing }
 ODBFileStreamWrapper >> position: anInteger [ 
-	"Positiones stream to anInteger. Answer anInteger."
```
Comment (noha) [inline src/OmniBase/ODBFileStreamWrapper.class.st:215]: Here we need to make sure that there are now users for the return value of position: I removed several already but there might be some left

Comment (MarcusDenker) [reply src/OmniBase/ODBFileStreamWrapper.class.st:215]: I checked and the only user of a return is in the makeTransactionObject: methods, but there it calls ODBMemoryStreamWrapper>>#position: which already returns self now.

**Pull request merged**